mail/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>spring-cloud-stream-mail-parent</artifactId>
+	<packaging>pom</packaging>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<artifactId>spring-cloud-stream-app-starters</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
I'm new to this project and I'm not so familiar (don't like ?) with Maven, so I might will ask a lot of noisy questions which may be obvious or know somehow for the team around.
So, my apologies if I will be so noobie :smile: 

Well, I see that this implicit version is used everywhere in each application.
Can't we extract this value somehow in the external resource and don't have a hell in the future to jump around to fix this value after each release? 
Thanks